src/App.vue

```diff
@@ -24,7 +24,8 @@
       url: "",
       body: "",
       language: "",
-      response: undefined
+      response: undefined,
+      wasResponseSuccessful: false
```

Comment:
Is this flag necessary? Does `wasResponseSuccessful` mean anything different than `response === undefined`?

Reply:
Removed